docs/hoon/twig/wut-test/bar-or.md

```diff
@@ -10,13 +10,25 @@ sort: 9
 ## Expands to
 
 ```
+:loop
 :ifno  p  
```

Comment:
Yeah, I understand. That same distinct is true of any twig that shows a list unwrapping with `i.p` and `t.p`. I guess I only took my objection so far ...